core/src/date-input/date-input.stories.tsx

```diff
@@ -27,6 +27,37 @@ export const Primary = () => {
 	);
 };
 
+export const Disabled = () => {
+	const [date, setDate] = useState<null | Date>(() => new Date());
+	return (
+		<div style={{ width: 240 }}>
+			<DateInput
+				format={DateInput.formats.dmy}
+				value={date}
+				setValue={setDate}
+				disabled
+				icon
+			/>
+		</div>
+	);
+};
+
+export const MinMax = () => {
+	const [date, setDate] = useState<null | Date>(() => new Date());
+	return (
+		<div style={{ width: 240 }}>
+			<DateInput
+				format={DateInput.formats.dmy}
+				value={date}
+				setValue={setDate}
+				maxDate={today}
+				minDate={yesterday}
```

Comment:
This is not really a good example. The range should be better (something like today + 7 and today - 7)